src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -1691,18 +1720,57 @@ LibraryServiceClient.prototype.addLabel = function(request, options, callback) {
  * @param {function(?Error, ?Object)=} callback
  *   The function which will be called with the result of the API call.
  *
- *   The second parameter to the callback is an object representing [google.longrunning.Operation]{@link external:"google.longrunning.Operation"}.
+ *   The second parameter to the callback is a [gax.Operation]{@link https://googleapis.github.io/gax-nodejs/global.html#Operation} object.
```

Comment:
`Operation` is a class, therefore the URL would be like .../Operation.html, not under global.html (see http://googleapis.github.io/gax-nodejs/PageDescriptor.html for example).